tests/dragonfly/acl_family_test.py

```diff
@@ -225,3 +227,73 @@ async def test_acl_with_long_running_script(df_server):
 
     await client.close()
     await admin_client.close()
+
+
+SCRIPTS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.mark.asyncio
+@dfly_args(
+    {
+        "aclfile": os.environ.get(
+            "DRAGONFLY_PATH", os.path.join(SCRIPTS_DIR, "../../build-dbg/error.acl")
+        ),
+        "port": 1111,
+    }
+)
+async def test_bad_acl_file(df_local_factory):
+    scripts_dir = os.path.dirname(os.path.abspath(__file__))
+    acl = os.environ.get("DRAGONFLY_PATH", os.path.join(scripts_dir, "../../build-dbg/error.acl"))
+    df = df_local_factory.create()
+
+    df.start()
+
+    client = aioredis.Redis(port=df.port)
+
+    with pytest.raises(redis.exceptions.ResponseError):
+        await client.execute_command("ACL LOAD")
+
+    await client.close()
+
+
+@pytest.mark.asyncio
+@dfly_args(
+    {
+        "aclfile": os.environ.get(
+            "DRAGONFLY_PATH", os.path.join(SCRIPTS_DIR, "../../build-dbg/ok.acl")
+        ),
```

Comment:
I meant keeping it inside the pytests folder... It looks kinda fragile. What if its not run from build-dbg? What if I want to edit it? Then I have to search it

Reply:
Btw ok.acl is empty 😄

Reply:
> Btw ok.acl is empty 

yes it's meant to be like that.

Reply:
>  It looks kinda fragile. What if its not run from build-dbg?

It is fragile but we use the exact same thing to run the tests so I guess it's fine ? (See `df_factory` fixture) 

Any other recommendation ?

Reply:
Also you can set the env variable DRAGONFLY_PATH which overwrites this, so it's fine. We do the exact same thing in the CI as well:

 34         export DRAGONFLY_PATH="${GITHUB_WORKSPACE}/${{inputs.build-folder-name}}/${{inputs.dfly-executable}}"